portals/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/AsyncApiUI.jsx

```diff
@@ -62,9 +61,18 @@ export default function AsyncApiUI(props) {
     } = props;
     const { api } = useContext(ApiContext);
 
-    let initialEndpoint = URLs && URLs.http;
-    if (api.type === CONSTANTS.API_TYPES.WS) {
-        initialEndpoint = URLs && URLs.ws;
+    let initialEndpoint;
+    if (api.advertiseInfo && api.advertiseInfo.advertised) {
```

Comment:
simplify the code

Reply:
done from https://github.com/wso2/apim-apps/pull/130/commits/38e365e23c13eb98da349ca0e1f1618291ff9b14